crates/pop-cli/tests/parachain.rs

```diff
@@ -2,13 +2,13 @@
 
 use anyhow::Result;
 use assert_cmd::{cargo::cargo_bin, Command};
-use pop_common::templates::Template;
+use pop_common::{find_free_port, templates::Template};
 use pop_parachains::Parachain;
 use std::{fs, path::Path, process::Command as Cmd};
 use strum::VariantArray;
 use tokio::time::{sleep, Duration};
 
-/// Test the parachain lifecycle: new, build, up
+/// Test the parachain lifecycle: new, build, up, call
```

Comment:
With public docs it would be good to strive for quality sentences, meaning dots at the end.
```suggestion
/// Test the parachain lifecycle: new, build, up, call.
```

Reply:
https://github.com/r0gue-io/pop-cli/pull/316/commits/7d7df02deee57a437203a1de8f2dbfe52b38b4d7